### DISCUSSION TITLE
Could we add a new icon for WordPress links in the author's socials?

### DISCUSSION THREAD

**Discussion by ilg-ul** [facebook/docusaurus#10561] [Ideas]
I did some tests with the new socials in the `authors.yml` and they seem pretty nice.

For now I identified only **x**, **github** and **linkedin** with specific icons, the rest are rendered with a generic globe icon.

Would you agree to add a new icon for WordPress links? Possibly Blogger? Possibly other blog solutions?

**Comment by slorber** [comment]
We could add some pre-built icons for the most common platforms but the goal is not to be exhaustive and I doubt WordPress or Blogger are worth adding.

We made the system extensible so that you can register your own custom icons through swizzle

Just add your label/icon there: 

`theme/Blog/Components/Author/Socials/index.tsx`

```js
const SocialPlatformConfigs: Record<string, SocialPlatformConfig> = {
  twitter: {Icon: Twitter, label: 'Twitter'},
  github: {Icon: GitHub, label: 'GitHub'},
  stackoverflow: {Icon: StackOverflow, label: 'Stack Overflow'},
  linkedin: {Icon: LinkedIn, label: 'LinkedIn'},
  x: {Icon: X, label: 'X'},
};
```

**Reply by ilg-ul** [reply]
> I doubt WordPress or Blogger are worth adding.

Blogger might not be worth adding, but WordPress is still quite popular and many people use it for personal blogs.

> you can register your own custom icons through swizzle

Yeah, I can use swizzle, but any component swizzled is like a subscription to undefined maintenance efforts, since it requires permanent future attention to keep up with next versions... 

Adding the WordPress icon to Docusaurus would be a one time commit that would require no further maintenance.

**Reply by slorber** [reply]
> WordPress is still quite popular and many people use it for personal blogs.

The fact that WordPress is a popular platform doesn't change anything. Regular human beings don't care about the platform you use.

Similarly I host my personal website on Vercel using Docusaurus, and it has a 🌐 icon, not a 🔼 or 🦖 icon. 

I am happy to help reduce the maintenance effort but currently adding icons has a cost in terms of bundle size for everyone and maintenance for us, and honestly WordPress is the last icon I would add, unless there's clear user traction for it (ie not only you but at least 5 others).

We are more likely to add icons for those things that regular human beings (ie not devs) reference using sentences such as "this person has <platform>". My parents would never use "this user has a WordPress", however they might say someone has an Instagram, TikTok, newsletter, podcast, website. Hope it makes sense. 

If "this user has a WordPress" makes sense for your target audience (which I doubt even if the audience is developers), then it's a specific case and should be added with swizzle.

**Reply by ilg-ul** [reply]
Ok, until there will be more users interested in this, I'll stick to the globe icon.
